images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -24,9 +24,9 @@ package_to_version () {
 # look up a binary with which and return the debian package it belongs to
```

Comment:
comment specifies `which`, except you're changing it to `command -v`

Reply:
Thanks, will fix.